### ISSUE TITLE
iOS Sourcemaps are not uploaded using react native 0.75.[1-3]

### ISSUE THREAD

**Issue by skogby** [getsentry/sentry-react-native#4120]
**OS:**
- [ ]  Windows
- [X]  MacOS
- [ ]  Linux

**Platform:**
- [X]  iOS
- [ ]  Android

**SDK:**
- [X]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 0.38.6

**`react-native` version:** "0.75.3

**Are you using Expo?**
- [ ]  Yes
- [X]  No

**Are you using sentry.io or on-premise?**
- [X]  sentry.io (SaaS)
- [ ]  on-premise

**If you are using sentry.io, please post a link to your issue so we can take a look:**

https://github.com/getsentry/sentry-react-native/issues/4074

**Configuration:**

(`@sentry/react-native`)
```
Sentry.init({
  dsn: 'https://***.ingest.sentry.io/***',
  environment: config.ENV_NAME,
  release: config.VERSION,
  dist: config.TIMESTAMP,
  enableTracing: true,
  tracesSampleRate: 0.005,
  debug: false,
  integrations: (integrations) => {
    integrations.push(
      Sentry.reactNativeTracingIntegration({
        routingInstrumentation,
      }),
    );
    return integrations;
  },
});
```

---
**I have the following issue:**

Sourcemaps are not found and uploaded by sentry script for iOS using React-native 0.75.1 -> 0.75.3.
There is more info in this issue, but it seems like Sentry maintainers are not looking into closed issues:
https://github.com/getsentry/sentry-react-native/issues/4074

**Steps to reproduce:**
- Install react native 0.75.3
- Install sentry with wizzard

**Actual result:**

```
output: sentry-cli - No packager source map found in source map report, skipping copy of debug id to Hermes combined source map.
```

**Expected result:**

```
output: sentry-cli - Using Hermes bundle and combined source map.
output: sentry-cli - Processing react-native sourcemaps for Sentry upload.
output: sentry-cli - > Analyzing 2 sources
output: sentry-cli - > Rewriting sources
output: sentry-cli - > Adding source map references
output: sentry-cli - > Bundled 2 files for upload
output: sentry-cli - > Bundle ID: XYZ
output: sentry-cli - > Uploaded files to Sentry
output: sentry-cli - > File upload complete (processing pending on server)
output: sentry-cli - > Organization: ***
output: sentry-cli - > Project: mobile-app
output: sentry-cli - > Release: com.x.y.z@8.24.0+1726677224
output: sentry-cli - > Dist: 1726677224
output: sentry-cli - > Upload type: artifact bundle
```

**Comment by kahest** [comment]
@skogby thanks for reporting and the details, we're gonna investigate and follow up here.

Apologies for the other issue not receiving attention - the engineer working on that (who would've been notified even though it's closed) is currently OOO and will be back on Monday.

**Comment by krystofwoldrich** [comment]
Thank you @skogby for the message and also for the new issue as that notifies other Sentry devs. The comments on closed issues only notify people originally involved.

Regardless, I'm back from my time off and I'm looking into this.

**Comment by krystofwoldrich** [comment]
I've reproduce the issue.

**Comment by krystofwoldrich** [comment]
I've reopened the original issue (https://github.com/getsentry/sentry-react-native/issues/4074) as there is more context user. I'll close this one as duplicate.
